crates/transport_tx5/src/harness.rs

```diff
@@ -39,7 +39,9 @@ impl Tx5TransportTestHarness {
             auth_material,
             transport: Tx5TransportFactory::create(),
             ..kitsune2_core::default_test_builder()
-        };
+        }
+        .with_default_config()
```

Comment:
Had to be added in order to be able to use in the kitsune crate, otherwise, the coreSpace config would be missing.